common-lib/src/main/scala/com/gu/mediaservice/model/FileMetadata.scala

```diff
@@ -11,7 +11,9 @@ case class FileMetadata(
   xmp: Map[String, String]       = Map(),
   icc: Map[String, String]       = Map(),
   getty: Map[String, String]     = Map(),
-  colourModel: Option[String]    = None
+  colourModel: Option[String]    = None,
+  pngData: Map[String, String]   = Map()
```

Comment:
Does this have to be specific to PNG data? Could we split the data collected into more generically named chunks?